src/app/styles.ts

```diff
@@ -0,0 +1,14 @@
+import {StyleSheet} from 'react-native';
+
+export default StyleSheet.create({
+  mainContainer: {
+    backgroundColor: '#EAF6FA',
```

Comment:
You should declare all the colors with hexadecimals and in a file of constants in `src/constants/colors`